src/main/kotlin/frc/robot/subsystems/swerve/SwerveConstants.kt

```diff
@@ -107,43 +100,44 @@ object SwerveConstants {
     var MAX_OMEGA_VELOCITY = 0.0
     var VY_NOTE_DETECTION_CONTROLLER = PIDController(5.0, 0.0, 0.3)
 
-    val TURN_MAX_TOLERANCE = 3.0/360.0 //TODO: calibrate
+    const val TURN_MAX_TOLERANCE = 3.0 / 360.0 //TODO: calibrate
+    const val SKID_TOLERANCE = 0.3
+    const val COLLISION_TOLERANCE: Float = 1.8F // [G]
```

Comment:
Measures

Reply:
Only `COLLISION_TOLERANCE` has units and the Units class doesn't have G